build/commands/scripts/format.js

```diff
@@ -162,6 +187,7 @@ const runPrettierForFile = async (file, dryRun, options, ignorePath) => {
 }
 
 const runPrettier = async (files, dryRun) => {
+  console.log('run prettier for', files.length, 'files')
```

Comment:
do we want this log?

Reply:
I did add it intentionally: we still log `git cl format` call anyway. And without this log it looks like `git cl format` hangs